docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -1135,6 +1135,51 @@ In this example, if the `remove` processor does fail, then
 the array elements that have been processed thus far will
 be updated.
 
+
+Another example, using the attachment processor:
```

Comment:
Did you forget to move the example in `docs/plugins/ingest-attachment.asciidoc`?
Please do it within the same PR. 

And then change the link to `{plugins}/ingest-attachment-with-arrays.html[attachment processor documentation]`?